Assignment2.2.sql

```diff
@@ -0,0 +1,112 @@
+USE storefront;
+
+INSERT INTO user_type(type_id , type_name) VALUES(1,"admininstrator");
```

Comment:
If type_id in the user_type table is auto incremental then why we are giving it in the query.